integration/tests/cook/test_basic.py

```diff
@@ -79,7 +82,8 @@ def test_failing_submit(self):
         self.assertEqual('failed', job['instances'][0]['status'])
 
     def test_max_runtime_exceeded(self):
-        job_uuid, resp = self.submit_job(command='sleep 60', max_runtime=5000)
+        timeout_interval_seconds = self.settings()['task-constraints']['timeout-interval-minutes'] * 60
```

Comment:
Should we be more careful on setting this timeout, as we can end up sleeping for a really long time.